org.mwc.debrief.legacy/src/Debrief/Wrappers/TrackWrapper.java

```diff
@@ -1385,6 +1385,7 @@ public void clearPositions()
   @Override
   public final void closeMe()
   {
+    flushBounds();

```

Comment:
I _think_ we should always flush the cache _after_ making data changes - so this should appear at the end of this message.

Reply:
Oh, ok